FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,59 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			// to prevent memory leak
+			guard self != nil else { return }
```

Comment:
This comment can be confusing. Now is it as a comment for line 24 and I don't think that `guard self != nil else { return }` prevents memory leak.
In general there's no need to leave such comments as all the behaviour is documented either with properly written code or with properly written tests.
Please remove the comment
```suggestion
		client.get(from: url) { [weak self] result in
			guard self != nil else { return }
```